Source/CMakeLists.txt

```diff
@@ -34,7 +34,7 @@ if(TARGET_PLATFORM_ANDROID OR TARGET_PLATFORM_IOS OR BUILD_AOT_CACHE)
 	list(APPEND DEFINITIONS_LIST DISABLE_LOGGING=1)
 endif()
 
-if(TARGET_PLATFORM_UNIX AND NOT TARGET_PLATFORM_UNIX_AARCH64)
+if(TARGET_PLATFORM_UNIX AND NOT TARGET_PLATFORM_UNIX_ARM AND NOT TARGET_PLATFORM_UNIX_AARCH64)
```

Comment:
Did you define `TARGET_PLATFORM_UNIX_ARM` in [Header.cmake](https://github.com/jpd002/Play-Dependencies/blob/master/cmake-modules/Header.cmake)?